### ISSUE TITLE
visitMethodInvocation is never visited

### ISSUE THREAD

**Issue by nomisRev** [openrewrite/rewrite-kotlin#21]
Reproducible example: https://github.com/nomisRev/rewrite-arrow/pull/3

This PR attempts to search for the `ensure` method invocation, but `visitMethodInvocation` is never called.
The resulting assertion message is also quite confusing, see below. Inspecting the code it seems that `after` in [`Assertions.kotlin` on line 49](https://github.com/openrewrite/rewrite-kotlin/blob/045630d3e129443ca41b9eca1757b5684dc088fb/src/main/java/org/openrewrite/kotlin/Assertions.java#L49) is never used, and discarded. 

```text
org.opentest4j.AssertionFailedError: [When parsing and printing the source code back to text without modifications, the printed source didn't match the original source code. This means there is a bug in the parser implementation itself. Please open an issue to report this, providing a sample of the code that generated this error!] 
expected: 
  "package com.yourorg
  
  import arrow.core.continuations.EffectScope
  
  fun EffectScope<String>.test(): Int {
    ensure(false) { "failure" }
    return 1
  }"
 but was: 
  "package com.yourorg
  
  import arrow.core.continuations.EffectScope
  
  fun EffectScope<String>.test(): Int {
    ensure(false,) { "failure")
    return 1
  }"
```

but actually expected:

```text
org.opentest4j.AssertionFailedError: [When parsing and printing the source code back to text without modifications, the printed source didn't match the original source code. This means there is a bug in the parser implementation itself. Please open an issue to report this, providing a sample of the code that generated this error!] 
expected: 
  "package com.yourorg
  
  import arrow.core.continuations.EffectScope
  
  fun EffectScope<String>.test(): Int {
    ensure(false) { "failure" }
    return 1
  }"
 but was: 
  "package com.yourorg
   
   import arrow.core.raise.Raise
   import arrow.core.raise.ensure
   
   fun Raise<String>.test(): Int {
     ensure(false) { "failure" }
     return 1
   }"
```

**Comment by traceyyoshima** [comment]
Hi @nomisRev, I tested the recipe on your sample project with the latest snapshots of `rewrite-kotlin` and `rewrite-gradle-plugin` published locally and the recipe runs. :)

Thanks for pioneering rewrite-kotlin recipe uses and reporting the issues.

**Comment by nomisRev** [comment]
My pleasure, thanks for working on such an awesome framework for facilitating code rewrites 🙌

**Comment by nomisRev** [comment]
Did you run this with `0.2.0-SNAPSHOT` or should I be using a different version? 🤔

**Comment by traceyyoshima** [comment]
> My pleasure, thanks for working on such an awesome framework for facilitating code rewrites 🙌

I'm happy to help!

> Did you run this with `0.2.0-SNAPSHOT` or should I be using a different version? 🤔

I did a point release of `rewrite-kotlin`, so you may update the version to use `0.2.1`. :)

**Comment by nomisRev** [comment]
Hey @traceyyoshima,
This is indeed fixed in 0.2.1. Shall we close this issue?
